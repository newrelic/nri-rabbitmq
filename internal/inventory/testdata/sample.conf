

# comment at beginning of line
	# tab indented comment
    # space indented comment
invalid
empty=
a=
b=c
=invalid
name1=value1 # after comment
  name2=value2  
  name3  =value3	# more comments
	name4  =	value4	
	name5.sub!more-text   = value5
name6=value6=more

name7  =    val7