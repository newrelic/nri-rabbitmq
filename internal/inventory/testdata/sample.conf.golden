{"config/a":{"value":""},"config/b":{"value":"c"},"config/empty":{"value":""},"config/name1":{"value":"value1"},"config/name2":{"value":"value2"},"config/name3":{"value":"value3"},"config/name4":{"value":"value4"},"config/name5.sub!more-text":{"value":"value5"},"config/name6":{"value":"value6=more"},"config/name7":{"value":"val7"},"config/nodeName":{"value":"node1"}}